### PR TITLE
Add batch progress section to gift entry batch mode

### PR DESCRIPTION
**Pull request by kenlewis-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by kenlewis-sfdo** [comment]
**lurch: attach W-037478

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HoXBQA0" target="_blank">W-037478</a>

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -5292,6 +5292,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Alt text: lightning-spinner</shortDescription>
         <value>Content is loading.</value>
     </labels>
+    <labels>
+        <fullName>geBatchGiftsCount</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Count of Gifts</shortDescription>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels:5300]:
```suggestion
        <shortDescription>Label for displaying the number of gifts in the batch</shortDescription>
```

**Comment by kenlewis-sfdo** [comment]
Replaced by this PR, which includes both the batch progress section and the functionality to enforce require total match: https://github.com/SalesforceFoundation/NPSP/pull/5494

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -5292,6 +5292,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
...
+        <value>Count of Gifts</value>
+    </labels>
+    <labels>
+        <fullName>geBatchGiftsTotal</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Total Batch Amount</shortDescription>
```
[the remark this answers is not in the document]
Comment (kenlewis-sfdo) [reply src/labels/CustomLabels.labels:5308]: Thanks @melissabarber ! I made these updates in the PR I opened to supersede this one: https://github.com/SalesforceFoundation/NPSP/pull/5494

**Comment by kenlewis-sfdo** [comment]
**lurch: attach W-038938

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HrByQAK" target="_blank">W-038938</a>

**Pull request closed without merging**